packages/app-mobile/components/NoteEditor/NoteEditor.tsx

```
@@ -468,10 +468,10 @@ function NoteEditor(props: Props) {
 				selectionState={selectionState}
 			/>
 			<View style={{
-				flexGrow: 1,
-				flexShrink: 0,
-				minHeight: '30%',
-			}}>
+					flexGrow: 1,
```

ESLint

[error] Expected indentation of 4 tabs but found 5.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 4 tabs but found 5.

(indent)